mods/modchooser/mod.yaml

```diff
@@ -4,6 +4,8 @@ Metadata:
 	Author: The OpenRA Developers
 	Hidden: true
 
+RequiresMods:
```

Comment:
Nor is it a good idea to have N required fields that modders don't know are required.